hassio/src/dashboard/hassio-update.ts

```diff
@@ -136,7 +136,7 @@ export class HassioUpdate extends LitElement {
   private async _confirmUpdate(ev): Promise<void> {
     const item = ev.currentTarget;
     if (item.key === "core") {
-      showDialogSupervisorCoreUpdate(this, { core: this.supervisor.core });
+      showDialogSupervisorCoreUpdate(this, { supervisor: this.supervisor });
```

Comment:
This param will not be updated if `supervisor` changed, not sure you are aware of that?

Reply:
It will still catch some of the cases, but those should be handled before you even open it I guess, I'll adjust tomorrow